joomla/joomla-cms
# [5.1] Fix radio and checkboxes in nested subform field

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/40160

### Summary of Changes

Fix radio and checkboxes in nested subform field


### Testing Instructions

Run `npm install`.

Add following XML for subform field, somewhere, example in Custom HTML module.
And check that Radio buttons and Checkboxes works.

```xml
<field type="subform" name="subform" label="subform" multiple="true">
    <form>
        <field type="text" name="text" label="Text" />
        <field name="radioTop" type="radio" label="Radio top">
            <option value="0">JNO</option>
            <option value="1">JYES</option>
        </field>
        <field name="check" type="checkboxes" label="Checkboxes">
            <option value="0">JNO</option>
            <option value="1">JYES</option>
        </field>
        <field name="checkOne" type="checkbox" label="Checkbox"/>

        <field type="subform" name="nested_subform" label="nested subform" multiple="true">
            <form>
                <field type="text" name="nested_text" label="Nested Text" />
                <field name="radioNested" type="radio" label="Nested Radio">
                    <option value="0">JNO</option>
                    <option value="1">JYES</option>
                </field>
                <field name="checkNested" type="checkboxes" label="Nested checkboxes">
                    <option value="0">JNO</option>
                    <option value="1">JYES</option>
                </field>
                <field name="checkOneNested" type="checkbox" label="Nested Checkbox"/>
            </form>
        </field>
    </form>
</field>
```



### Actual result BEFORE applying this Pull Request
Radio buttons and checkboxes in nested subform does not get checked when click on its label.


### Expected result AFTER applying this Pull Request
All radio buttons and checkboxes works.


### Link to documentations
Please select:
